library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/impl/ResourceLoaderImpl.java

```diff
@@ -472,23 +472,23 @@ private static boolean registerBuiltinResourcePack(ResourceType type, ModNioReso
 		return false;
 	}
 
-	private static ModNioResourcePack newBuiltinResourcePack(ModContainer container, String name, Text displayName,
-			Path resourcePackPath, ResourceType type, ResourcePackActivationType activationType) {
-		return new ModNioResourcePack(name, container.metadata(), displayName, activationType, resourcePackPath, type, null);
+	private static ModNioPack newBuiltinResourcePack(ModContainer container, String name, Text displayName,
+													 Path resourcePackPath, ResourceType type, PackActivationType activationType) {
```

Comment:
This indentation change might be worth to check with the style conventions, but it may also just be worth ignoring.